packages/core/core/src/AssetGraph.js

```diff
@@ -317,4 +318,8 @@ export default class AssetGraph extends Graph<AssetGraphNode> {
       value: asset
     });
   }
+
+  _dumpToGraphViz(name: string): Promise<string> {
```

Comment:
Shouldn't this become part of the reporter instead of core?

Reply:
If it were part of the reporter, we'd have to expose the raw graph as part of public API, which I don't think we want to do. I agree this isn't optimal. I'm open to suggestions 😄

Reply:
Ow thought we were gonna sent a minimal graph to the reporter that was standardised. My bad.

Reply:
We do exactly that 😄 : https://github.com/parcel-bundler/parcel/blob/d7e740e46f1a8de564a7706159772ff9d4e00640/packages/core/core/src/Parcel.js#L128

However, this reporter needs access to the raw graph 😞

Reply:
Ow alright just thought the public api graph was more complete then.
Guess I'll have to look a little bit more into how minimal it actually is.